wren-modeling-rs/README.md

```diff
@@ -10,7 +11,25 @@ There are some modules:
 # How to Test / Build
 
 - Run Tset
-Currently, the test cases are placed in `src/mdl/mod.rs`
+  Currently, the test cases are placed in `src/mdl/mod.rs`
+
 ```
 cargo test
 ```
+
+# Coding Style
+
+Please format your code with `rustfmt` and `taplo` before submitting a pull request.
+
+## Format with rustfmt
+
+```
+cargo fmt
+```
+
+## Format with taplo
```

Comment:
IMO, the taplo is only for toml. How about add toml to this line like `Format toml with taplo` to distinguish from rustfmt.